### PR TITLE
updated configuration

### PR DESCRIPTION
**Pull request by ghost**
The laser calibration is fine now; however, the camera calibration does not work; we always get those weird values that put the pan/tilt mounting somewhere up to the third link of the arm; we followed the procedure in detail, however, it just does not work. Either we try it again on Monday or put there some approximated values.

### CODE REVIEW

File: robotino_bringup/robots/uibk-robotino2/launch/uibk-robotino2.launch [hunk 1]
```diff
@@ -10,10 +10,10 @@
 
   <!-- COMMON -->
   <!-- send robot urdf to param server -->
-  <param name="robot_description" command="$(find xacro)/xacro.py '$(find robotino_bringup)/robots/$(arg robot)/urdf/robotino.urdf.xacro'" />
+  <param name="robot_description" command="$(find xacro)/xacro.py '$(find robotino_bringup)/robots/$(arg robot)/urdf/$(arg robot).urdf.xacro'" />
```
Comment (NadiaHG) [inline robotino_bringup/robots/uibk-robotino2/launch/uibk-robotino2.launch:13]: Does it work? the name of the urdf file is robotino.urdf.xacro

File: robotino_bringup/robots/uibk-robotino2/launch/uibk-robotino2.launch [hunk 2]
```diff
@@ -80,11 +80,11 @@
 
   <!-- Simple node to move the kinect in the default position on robot's startup
        It also exposes a service /tilt_controller/resetPosition -->
-  <!--node name="robotino_tilt_controller" pkg="robotino_driver" type="robotino_tilt_controller" output="screen">
+  <node name="robotino_tilt_controller" pkg="robotino_driver" type="robotino_tilt_controller" output="screen">
```
Comment (NadiaHG) [inline robotino_bringup/robots/uibk-robotino2/launch/uibk-robotino2.launch:83]: your robot has only a tilt axis, or a pan+tilt?

**Comment by ghost** [comment]
Hi, can you close and reject the request? We have to do it manually, both calibration routines don't work and just spit out bogus values.

**Comment by ghost** [comment]
Ok, we again redid it and now it seemed to have worked; we do one final test with the real robot and create a new pull request in case.

**Comment by ghost** [comment]
Still wrong values. Just close and reject. No calibration for now.

**Pull request closed without merging**